packages/block-editor/src/components/date-format-picker/index.js

```diff
@@ -76,24 +82,9 @@ function NonDefaultControls( { format, onChange } ) {
 	// 2022) in German (de). The resultant array is de-duplicated as some
 	// languages will use the same format string for short, medium, and long
 	// formats.
-	const suggestedFormats = [
-		...new Set( [
-			/* translators: See https://www.php.net/manual/datetime.format.php */
-			'Y-m-d',
-			/* translators: See https://www.php.net/manual/datetime.format.php */
-			_x( 'n/j/Y', 'short date format' ),
-			/* translators: See https://www.php.net/manual/datetime.format.php */
-			_x( 'n/j/Y g:i A', 'short date format with time' ),
-			/* translators: See https://www.php.net/manual/datetime.format.php */
-			_x( 'M j, Y', 'medium date format' ),
-			/* translators: See https://www.php.net/manual/datetime.format.php */
-			_x( 'M j, Y g:i A', 'medium date format with time' ),
-			/* translators: See https://www.php.net/manual/datetime.format.php */
-			_x( 'F j, Y', 'long date format' ),
-			/* translators: See https://www.php.net/manual/datetime.format.php */
-			_x( 'M j', 'short date format without the year' ),
-		] ),
-	];
+	const editorSettings = select( blockEditorStore ).getSettings();
```

Comment:
@TylerB24890 can you refactor this to use `useSelect` instead? There are some architectural differences and the usage of select itself is discouraged

Reply:
All set.